steemapi/steemnoderpc.py

```diff
@@ -174,3 +174,28 @@ def stream(self, opNames, *args, **kwargs):
                 for op in tx["operations"]:
                     if op[0] in opNames:
                         yield op[1]
+
+    def list_accounts(self, start=None, step=1000, limit=None):
+        """ Yield list of user accounts in alphabetical order
+
+        :param str start: Name of account, which should be yield first
+        :param int step: Describes how many accounts should be fetched in each rpc request
+        :param int limit: Limit number of returned user accounts
+        """
+        if limit and limit < step:
+            step = limit
+
+        fetched_users = 0
+        progress = step
+
+        while progress == step and (not limit or fetched_users < limit):
+            users = self.lookup_accounts(start, step)
+            progress = len(users)
+
+            if progress > 0:
+                yield from users
+                fetched_users += progress
+
+                # concatenate last fetched account name with lowest possible
+                # ascii character to get next lowest possible login as lower_bound
+                start = users[-1] + '\0'
```

Comment:
[This code](https://github.com/xeroc/python-steemlib/commit/9008e17aa8299b29bbe751ef98cdf4f0e360b90a#diff-a1cf39198f474daa3081bc11527ffe77R76) shows how I am doing it with account history.

Why do you keep 'fetch_users'? and not just see if `users` has `step` elements?